app/workers/asset_manager_create_asset_worker.rb

```diff
@@ -6,78 +6,25 @@ class AssetManagerCreateAssetWorker < WorkerBase
   # Use TransactionAwareClient for this worker to ensure that the commit is finished before the worker is executed.
   sidekiq_options queue: "asset_manager", client_class: Sidekiq::TransactionAwareClient
 
-  def perform(temporary_location, asset_params, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [])
+  def perform(temporary_location, asset_params, auth_bypass_ids = [])
```

Comment:
I think we're going to have to make the changes to the signature in a different PR, or rather, across several PRs, otherwise there are going to be `ArgumentError`s. There will be jobs of the old signature queued up when we release the new one.  

If I remember correctly, the way we typically address this is by doing this painful staggered PRs dance, where we create the new worker, release, allow for time to empty the queues of the old worker, then delete the old worker. In case we want to still use the old worker name, then even more dancing is required. 
[This](https://keygen.sh/blog/how-to-change-the-signature-of-a-sidekiq-job/) sums it up nicely.

Reply:
Oh I love migrations